example_simulation/simulation_config_materialsproject.conf

```diff
@@ -5,7 +5,7 @@
 # formulas to be queried from Materialsproject. mp_Fe-S will cause all
 # materials with iron and sulfur to be downloaded and a worker json
 # configuration will be generated for each material.
-material = [gold_fcc,mp_Ar,mp_Ne]
+material = [gold_fcc,mp_C-O,mp_Fe-S]
```

Comment:
Should this change really included in this PR?

Reply:
done